README.md

```diff
@@ -32,6 +32,23 @@ To use the module in your code, instantiate one of the following:
 * ```new Talis\Persona\Client\Users``` - for user based Persona calls
 * ```new Talis\Persona\Client\OAuthClients``` - for oauth based Persona calls
 
+### Versioning
+
+The client by default will use the latest Persona API version. 
+This is defined in: `\Talis\Persona\Client\Base::PERSONA_API_VERSION`
+ 
+To override the version used, provide a `persona_api_version` config option e.g:

```

Comment:
I would vote not to include the option of overriding the version. The version should be fixed within the client. There could be a misalignment between the client and the version of the API.